draft-irtf-cfrg-frost.md

```diff
@@ -1530,12 +1546,12 @@ result. If it fails, try again with another random byte array, until the
 procedure succeeds. Failure to implement `DeserializeScalar` in constant time
 can leak information about the underlying corresponding Scalar.
 
-Note the that the Scalar size might be some bits smaller than the array size,
-which can result in the loop iterating more times than required. In that case
-it's acceptable to set the high-order bits to 0 before calling `DeserializeScalar`,
-but care must be taken to not set to zero more bits than required. For example,
-in the `FROST(Ed25519, SHA-512)` ciphersuite, the order has 253 bits while
-the array has 256; thus the top 3 bits of the last byte can be set to zero.
+As an optimization, if the group order is very close to a power of
+2, it is acceptable to omit the rejection test completely.  In
+particular, if the group order is p, and there is an integer b
+such that `p - 2<sup>b</sup>| < 2<sup>(b/2)</sup>`, then
+`RandomScalar` can simply return a uniformly random integer of at
+most b bits.
```

Comment:
Just to confirm, was the intention to actually replace the old text? These seem unrelated and we could arguably include both paragraphs. I don't mind either way though.

Reply:
Same question here.

Reply:
Yeah, the intention was to replace the old text with this. The old text didn't apply to all suites and assumed too much of the internals of DeserializeScalar. This new text avoids both problems.